include/swift/AST/DiagnosticsParse.def

```diff
@@ -1317,6 +1317,11 @@ ERROR(attr_objc_expected_rparen,none,
       "expected ')' after name for @objc", ())
 ERROR(attr_objc_empty_name,none,
       "expected name within parentheses of @objc attribute", ())
+ERROR(attr_nskeyedarchiverclassname_removed, none,
```

Comment:
I plan to take them out later. I left a note in the actual parser but probably should have done one here as well.

(I agree, it's very unlikely anyone is using them.)